src/libraries/System.Drawing.Common/ref/System.Drawing.Common.netcoreapp.cs

```diff
@@ -19,13 +19,46 @@ public sealed partial class Graphics
         public void GetContextInfo(out PointF offset) { throw null; }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public void GetContextInfo(out PointF offset, out Region? clip) { throw null; }
+        public System.IntPtr Handle { get { throw null; } }
+        public static System.Drawing.Graphics FromHandle(System.IntPtr handle) { throw null; }
+    }
+
```

Comment:
Our goal is to either generate these files (as discussed earlier) or make it look like the generator created it. The generator won't insert blank lines like this as we would in normal code.